M2/Macaulay2/packages/SimpleDoc/doc.txt

```diff
@@ -1,192 +1,263 @@
-  Item
-   The string {\tt s} is handled by parsing it the same way @ TO doc @ does,
-   but expecting the least indented keyword to be {\bf Node}, which can occur
-   multiple times.  The text following each such keyword is handled just as by
-   @ TO doc @, resulting in a documentation node.
- Description
-  Text
-   Here is the input submitted to @ TO multidoc @ to document (most of) this package:
-  Code
-   EXAMPLE { PRE concatenate between("\n", lines get(SimpleDoc#"source directory" | "SimpleDoc/doc.txt")) }
+  Key
+    SimpleDoc
+  Headline
+    simpler documentation for functions and methods
+  Description
+    Text
+      This package provides a human-readable serialization format for documentation nodes
+      as an alternative to the classic way of @TO "Macaulay2Doc::writing documentation"@
+      using the @TO document@ method.
+
+      The new format is parsed by @TO (doc, String)@ for writing either a single or
+      multiple documentation nodes from string or file and adheres to the off-side rule;
+      i.e., using indentation to separate blocks.
+    Tree
+      :Getting started with SimpleDoc
+        @TO2 {doc, "Using SimpleDoc for documentation"}@
+        @TO2 {"Text", "Using markup in documentation"}@
+        @TO2 {"Text::html(TEX)", "Using TeX in documentation"}@
+        @TO2 {"Macaulay2Doc::conventions for documentation", "Style guide for documentation"}@
+
+      :Documentation helpers
+        arXiv
+        stacksProject
+        wikipedia
+  Contributors
+    Andrew Hoefel contributed to this package.
 Node
   Key
-    packageTemplate
-    (packageTemplate,String)
+     packageTemplate
+    (packageTemplate, String)
   Headline
     a template for a package
   Usage
-    packageTemplate s
+    packageTemplate name
   Inputs
-    s:String
+    name:String
       the name of the package
   Description
    Text
-     This routine returns a barebones package template
-     that you can use to start writing a package.
+     This routine returns a barebones package template that can be used to start writing a package.
    Example
      print packageTemplate "WonderfulModules"
   SeeAlso
     "docExample"
+    "testExample"
 Node
   Key
-    "docTemplate"
+    "testExample"
   Headline
-    a template for a documentation node
+    an example testcase
   Usage
-    docTemplate
+    testExample
   Description
    Text
-     docTemplate is a @TO String@, which can
-     be cut and paste into a text file,
-     to be used as a template for writing
-     documentation for functions and other objects
-     in a package.
-   Example
-     print docTemplate
-  SeeAlso
-    "docExample"
-Node
-  Key
-    simpleDocFrob
-  Headline
-    an example of a function to document
-Node
-  Key
-    "docExample"
-  Headline
-    an example documentation node
-  Usage
-    docExample
-  Description
+     The variable {\tt testExample} is a @TO String@ containing an example of
+     the use of @TO (TEST, String)@ to write a test case.
```

Comment:
To get `TO (TEST, String)` to work, you have to direct it to the Macaulay2Doc package, which is where that documentation node is.